### PR TITLE
Fix animation bug

### LINKED ISSUE #4312: Animated.Value does not increment properly
Say for example I want a value to animate between 0 and 1. Nothing would happen if I were to do this. I must set the initial value to >0.001 for it to actually do anything.

### Environment
react-native-cli: 2.0.1
react-native: 0.61.5
  System:
    OS: Windows 10 10.0.18363
    CPU: (12) x64 Intel(R) Xeon(R) W-2133 CPU @ 3.60GHz
    Memory: 15.79 GB / 31.73 GB
  Binaries:
    Node: 10.16.3 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
  npmPackages:
    react: 16.9.0 => 16.9.0
    react-native: ^0.61.5 => 0.61.5
    react-native-windows: 0.61.0-beta.22 => 0.61.0-beta.44
  Installed UWP SDKs:
    10.0.17763.0
    10.0.18362.0

### Steps to Reproduce

Use this app code:
https://snack.expo.io/I5ZIbQtHJ

### Expected Behavior

The opacity of the text shifts from 0 to 1 over the course of 1 second.

### Actual Behavior

The Text element stays invisible forever. However, if I initialize mySmallAnimatedValue to any value > 0.001, it will work properly.

### Additional context

Adam Gorman(adamgor)
Austin Beaulieu(aubeauli)

### PR DESCRIPTION
**Pull request by kmelmon**
Fixes #4312 

xbox folks ran into this animation bug where the component stays stuck invisible.  I root-caused this back to a dirty flag issue in XAML.  I filed this bug for us to fix the XAML bug in WinUI 3.0:
https://github.com/microsoft/microsoft-ui-xaml/issues/2511

Meanwhile this change works around the XAML bug in a very non-invasive way.  We can force the XAML element to render by "poking" one of the properties that dirties it for render (then set it back to whatever it was).  Once dirtied, the XAML render walk will put the visuals back into the tree and the element will become visible again.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/4964)

### CODE REVIEW

**Comment by asklar** [comment]
> Fixes #4312
> 
> xbox folks ran into this animation bug where the component stays stuck invisible. I root-caused this back to a dirty flag issue in XAML. I filed this bug for us to fix the XAML bug in WinUI 3.0:
> [microsoft/microsoft-ui-xaml#2511](https://github.com/microsoft/microsoft-ui-xaml/issues/2511)
> 
> Meanwhile this change works around the XAML bug in a very non-invasive way. We can force the XAML element to render by "poking" one of the properties that dirties it for render (then set it back to whatever it was). Once dirtied, the XAML render walk will put the visuals back into the tree and the element will become visible again.
> 
> ###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/4964)

cool! :)

File: vnext/ReactUWP/Modules/Animated/PropsAnimatedNode.cpp [hunk 1]
```diff
@@ -97,9 +97,25 @@ void PropsAnimatedNode::UpdateView() {
   StartAnimations();
 }
 
+static void EnsureUIElementDirtyForRender(xaml::UIElement uiElement) {
+  auto compositeMode = uiElement.CompositeMode();
```
Comment (asklar) [inline vnext/ReactUWP/Modules/Animated/PropsAnimatedNode.cpp:101]: can make const

**Comment by NickGerleman** [comment]
Adding a backport tag since we had previously discussed this as a candidate.

**Comment by Khalef1** [comment]
Removing Request Backport and Backport to 0.62 label. Based on our triage discussion, we won't be backporting this change.

**Pull request merged**